clients/java/src/main/java/io/camunda/client/api/search/enums/ProcessInstanceState.java

```diff
@@ -19,6 +19,5 @@ public enum ProcessInstanceState {
   ACTIVE,
   COMPLETED,
   TERMINATED,
-  SUSPENDED,
```

Comment:
@korthout I introduced a new state `INTERRUPTED_BY_RUNTIME_INSTRUCTION` in this PR, but since it's a transient state that is immediately replaced by `TERMINATED`, I thought it doesn't make sense to expose it to clients. Any thoughts on this?